.github/workflows/update-swagger-ghpages.yml

```diff
@@ -0,0 +1,28 @@
+name: update-swagger-ghpages
+
+on:
+  push:
+    tags:
+      - v*
+    branches: [ main ]
+  pull_request:
+jobs:
+  build-documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: gh-pages
+      - name: get swagger.yaml
+        run: |
+          curl https://raw.githubusercontent.com/sapcc/archer/main/swagger.yaml -o swagger.yaml
```

Comment:
This would mean that when a release is tagged the latest swagger from main is taken, not from the release tag which could include already newer changes.

Reply:
Correct, this was just easier to implement, or do you propose a change ;)

Reply:
Clone the repo and use the file from there? That pretty standard for gh actions.